typescript/infra/src/deploy.ts

```diff
@@ -67,14 +67,14 @@ export async function deployWithArtifacts<Config>(
   },
 ) {
   if (cache.read) {
-    let addresses = {};
+    let addressesMap = {};
```

Comment:
If we're renaming for clarity, I find `xToY` is a helpful nomenclature for maps

Reply:
Agreed but prefer thsi for now as it's more consistent with how we've named things elsewhere (`xMap` as implicitly `chainToX`)